integration_test/functions/src/index.ts

```diff
@@ -1,83 +1,116 @@
 import { PubSub } from '@google-cloud/pubsub';
 import { Request, Response } from 'express';
+import fetch from 'node-fetch';
 import * as admin from 'firebase-admin';
 import * as functions from 'firebase-functions';
 import * as fs from 'fs';
-import * as https from 'https';
 
-export * from './pubsub-tests';
-export * from './database-tests';
-export * from './auth-tests';
-export * from './firestore-tests';
-export * from './https-tests';
-export * from './remoteConfig-tests';
-export * from './storage-tests';
-export * from './testLab-tests';
-const numTests = Object.keys(exports).length; // Assumption: every exported function is its own test.
+import * as v1 from './v1/index';
+const numTests = Object.keys(v1).length; // Assumption: every exported function is its own test.
```

Comment:
If you wanted to avoid this assumption you could technically filter for those functions which had `__trigger` or `__endpoint`